earth_enterprise/BUILD.md

```diff
@@ -0,0 +1,57 @@
+If you want to build, install, or run Portable server, see the [Portable Server page](https://github.com/google/earthenterprise/wiki/Portable-Server).
+
+# Building Earth Enterprise Fusion and Server
+
+Building is currently supported on 64-bit versions of Ubuntu 14.04 LTS, Ubuntu 16.04 LTS, RHEL 6, RHEL 7, and CentOS 7.
+
+
+## GEE 5.2.1 Build Prerequisites (All platforms) 
+
+1. Setup required Tools and Dependencies
+    * git 1.7.1+
+    * git lfs 
+    * gcc 4.8.x
+    * scons 2.0.x+
+    * python 2.6.x+ and Python 2.7.x+.  Python 3.0+ is not supported. 
+    
+    Different operating systems have different means to setup up these dependencies. For Linux build environments, see either the [Redhat and Centos Setup Instructions](./BUILD_RHEL_Centos.md)  
```

Comment:
Are these intentionally on separate lines or is this meant to be a single paragraph?  If you want them to be on the same line you need to get rid of the spaces at the end of the first line.

Reply:
Separate paragraph - but I removed the leading spaces anyway - the remaining line break keeps the paragraph separated properly.